### DISCUSSION TITLE
Dosbian 2.5 switched from DOSBox-X to DOSBox Staging

### DISCUSSION THREAD

**Discussion by Grounded0** [dosbox-staging/dosbox-staging#3788] [General]
https://cmaiolino.wordpress.com/dosbian/

"Added the wonderful Dosbox Staging 0.81"
"Removed Dosbox-X"

It's nice when people give high praise to our releases like this.

**Comment by dreamer** [comment]
Yup, I felt warm and fuzzy when I read that. Their description also says "a modern Dosbox, best experience ever for DOS games".

**Reply by johnnovak** [reply]
Noice 😎 Thanks for posting @Grounded0 👍🏻

**Reply by johnnovak** [reply]
We might wanna call this out in the 0.82.0 release notes 😎 🤘🏻
